symfony/plugins/orangehrmCorePlugin/test/Service/DateTimeHelperServiceTest.php

```diff
@@ -216,4 +216,11 @@ public function testGetNow()
         $this->assertEquals((new DateTime())->getTimezone(), $this->dateTimeHelperService->getNow()->getTimezone());
         $this->assertEquals((new DateTime())->getOffset(), $this->dateTimeHelperService->getNow()->getOffset());
     }
+
+    public function testGetTimezoneByTimezoneOffset(): void
+    {
+        $this->assertEquals($this->dateTimeHelperService->getTimezoneByTimezoneOffset(5.5), new DateTimeZone('+0530'));
+        $this->assertEquals($this->dateTimeHelperService->getTimezoneByTimezoneOffset(-5.5), new DateTimeZone('-0530'));
+        $this->assertEquals($this->dateTimeHelperService->getTimezoneByTimezoneOffset(5.0), new DateTimeZone('+0500'));
```

Comment:
`$this->assertEquals($expected, $actual)` @vageeshorangehrm am I wrong?
```suggestion
        $this->assertEquals(new DateTimeZone('+0500'), $this->dateTimeHelperService->getTimezoneByTimezoneOffset(5.0));
```

Reply:
public static function assertEquals($expected, $actual, string $message = ''): void
    {
        $constraint = new IsEqual($expected);

        static::assertThat($actual, $constraint, $message);
    }

Reply:
sorry I have changed the direction

Reply:
no rajitha you are correct, first argument is expected and second one is actual out put